thirdeye-ui/src/app/components/rca/heat-map-section/heat-map-section.component.tsx

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright 2023 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import { Box, Button, Grid, Typography } from "@material-ui/core";
+import { every, isEmpty } from "lodash";
+import React, { FunctionComponent, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useOutletContext, useParams, useSearchParams } from "react-router-dom";
+import { EmptyStateSwitch } from "../../page-states/empty-state-switch/empty-state-switch.component";
+import { LoadingErrorStateSwitch } from "../../page-states/loading-error-state-switch/loading-error-state-switch.component";
+import { AnomalyFilterOption } from "../anomaly-dimension-analysis/anomaly-dimension-analysis.interfaces";
+import { DimensionSearchAutocomplete } from "../heat-map/dimension-search-automcomplete/dimension-search-autocomplete.component";
+import { HeatMap } from "../heat-map/heat-map.component";
+import { formatDimensionOptions } from "../heat-map/heat-map.utils";
+import { PreviewChart } from "../top-contributors-table/preview-chart/preview-chart.component";
+import {
+    PageContentsCardV1,
+    SkeletonV1,
+    useNotificationProviderV1,
+} from "../../../platform/components";
+import { ActionStatus } from "../../../rest/actions.interfaces";
+import {
+    AnomalyBreakdown,
+    SavedStateKeys,
+} from "../../../rest/dto/rca.interfaces";
+import { useGetAnomalyMetricBreakdown } from "../../../rest/rca/rca.actions";
+import { areFiltersEqual } from "../../../utils/anomaly-dimension-analysis/anomaly-dimension-analysis";
+import { getFromSavedInvestigationOrDefault } from "../../../utils/investigation/investigation.util";
+import { notifyIfErrors } from "../../../utils/notifications/notifications.util";
+import {
+    concatKeyValueWithEqual,
+    deserializeKeyValuePair,
+    serializeKeyValuePair,
+} from "../../../utils/params/params.util";
+import { RootCauseAnalysisForAnomalyPageParams } from "../../../pages/root-cause-analysis-for-anomaly-page/root-cause-analysis-for-anomaly-page.interfaces";
+import { InvestigationContext } from "../../../pages/rca/investigation-state-tracker-container-page/investigation-state-tracker.interfaces";
+import { HeatMapSectionProps } from "./heat-map-section.interfaces";
+
+const HEATMAP_FILTERS_URL_KEY = "heatmapFilters";
+
+export const HeatMapPage: FunctionComponent<HeatMapSectionProps> = ({
```

Comment:
This is the renamed [#1360 (files)](https://github.com/startreedata/thirdeye/pull/1360/files#diff-1c98ced186fa5a374cca0cca6048442290b48804c52809d50519fe691e157581) file that is shown as deleted below